components/partials/Header.js

```diff
@@ -0,0 +1,176 @@
+              </li>
+
+              <li className="my-2 mr-4 font-normal ">
+                <Link
+                  href="/subscribe"
+                  className={
+                    router.pathname == "/subscribe"
+                      ? "md:mr-6 xl:mr-9 relative pb-[5.5px] text-transparent bg-clip-text bg-gradient-to-r after:bg-gradient-to-r from-pink-300 to-orange-300 after:absolute after:bottom-0 after:left-0 after:w-full after:h-[2px]"
+                      : "md:mr-6 xl:mr-9 text-black-900 hover:relative pb-[5.5px] hover:text-transparent hover:bg-clip-text  hover:bg-gradient-to-r hover:after:bg-gradient-to-r from-pink-300 to-orange-300 hover:after:absolute hover:after:bottom-0 hover:after:left-0 hover:after:w-full hover:after:h-[2px]"
+                  }
+                >
+                  Subscribe
+                </Link>
+              </li>
+
+              <li className="my-2 mr-4 font-normal ">
+                <Link
+                  href="/write"
+                  className={
+                    router.pathname == "/write"
+                      ? "md:mr-6 xl:mr-9 relative pb-[5.5px] text-transparent bg-clip-text bg-gradient-to-r after:bg-gradient-to-r from-pink-300 to-orange-300 after:absolute after:bottom-0 after:left-0 after:w-full after:h-[2px]"
+                      : "md:mr-6 xl:mr-9 text-black-900 hover:relative pb-[5.5px] hover:text-transparent hover:bg-clip-text  hover:bg-gradient-to-r hover:after:bg-gradient-to-r from-pink-300 to-orange-300 hover:after:absolute hover:after:bottom-0 hover:after:left-0 hover:after:w-full hover:after:h-[2px]"
+                  }
+                >
+                  Write
+                </Link>
+              </li>
+
+              {/* ==== If User Not Login ====  */}
+
+              {/* <li className="my-2 font-normal ">
+                <Link
+                  href="/login"
+                  className={
+                    router.pathname == "/login"
+                      ? "md:mr-6 xl:mr-9 relative pb-[5.5px] text-transparent bg-clip-text bg-gradient-to-r after:bg-gradient-to-r from-pink-300 to-orange-300 after:absolute after:bottom-0 after:left-0 after:w-full after:h-[2px]"
+                      : "md:mr-6 xl:mr-9 text-black-900 hover:relative pb-[5.5px] hover:text-transparent hover:bg-clip-text  hover:bg-gradient-to-r hover:after:bg-gradient-to-r from-pink-300 to-orange-300 hover:after:absolute hover:after:bottom-0 hover:after:left-0 hover:after:w-full hover:after:h-[2px]"
+                  }
+                >
+                  Login
+                </Link>
+              </li>  */}
+
+              <li className="my-2 font-normal">
+                <div
+                  onClick={() => setdropdownOpen(!dropdownOpen)}
+                  className="items-center flex justify-center mr-10 md:mr-6 xl:mr-9 overflow-hidden text-black-900  
+                            hover:cursor-pointer"
+                >
+                  Profile
+                  <svg
```

Comment:
Can add a default profile image instead of SVG. Ex: Make circle div and fit profile image into it